### PR TITLE
DC statement

### PR DESCRIPTION
**Pull request by maelle**
needs review

### CODE REVIEW

File: content/post/statement-about-datacamp.Rmd [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+1. Cancel your subscription.
+
+2. Cancel your organization's subscription.
+
+3. Remove links/recommendations to use DataCamp from your material.
+
+4. Stop developing content/lectures for them. Folks who are not customers can alter behavior as well.
+
+5. take your DataCamp stickers off your laptop/car/water bottle/ etc.
```
Comment (ledell) [inline content/post/statement-about-datacamp.Rmd]:
```suggestion
5. Take your DataCamp stickers off your laptop/car/water bottle/ etc. 
```

File: content/post/statement-about-datacamp.Rmd [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+2. Cancel your organization's subscription.
+
+3. Remove links/recommendations to use DataCamp from your material.
+
+4. Stop developing content/lectures for them. Folks who are not customers can alter behavior as well.
+
+5. take your DataCamp stickers off your laptop/car/water bottle/ etc. 
+
+6. just say "No." when they ask you to do a podcast.
```
Comment (ledell) [inline content/post/statement-about-datacamp.Rmd]:
```suggestion
6. Just say "No" when they ask you to do a podcast.
```

File: content/post/statement-about-datacamp.Rmd [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+7. Don't accept #DataCamp as the sponsor of events you organize.
```
Comment (ledell) [inline content/post/statement-about-datacamp.Rmd]:
```suggestion
7. Don't accept DataCamp as a sponsor of events you organize.
```

File: content/post/statement-about-datacamp.Rmd [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+8. Delete your account (even if you don't have a paid subscription anymore).
+
+9. Remove your tweets containing “DataCamp" by searching `DataCamp from:@youruser` (replace `@youruser` by your own handle) in Twitter search and deleting the old tweets that endorsed they in any way.
+
```
Comment (ledell) [inline content/post/statement-about-datacamp.Rmd]:
```suggestion

10. If you are working on a course or project for DataCamp, you can put your work on hold until you feel that appropriate and sufficient action has been taken on behalf of DataCamp.

```

File: content/post/statement-about-datacamp.Rmd [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+We are not satisfied with the response from DataCamp. As underlined by [Julia Silge](https://twitter.com/juliasilge/status/1113996214561923072) and [Mara Averick](https://twitter.com/dataandme/status/1113939480300355584), the post used "details in victim-blaming ways".
+
+# How can you stand up for more accountability?
+
+If you are unsatisfied with their actions & response, you can use your voice and your wallet to vote for more accountability. Below we are summarizing ideas from a [thread started by Jesse Mostipak](https://twitter.com/kierisi/status/1113945904698892289), with contributions from [Chris Engelhardt](https://twitter.com/EngelhardtCR), [Laura Ación](https://twitter.com/_lacion_/status/1113960425174052864), [Bruna Wunderwald](https://twitter.com/bwundervald/status/1114907322441244672).
```
Comment (ledell) [inline content/post/statement-about-datacamp.Rmd]:
```suggestion
If you are unsatisfied with their actions & response, you can use your voice and your wallet to vote for more accountability. Below we are summarizing ideas from a [thread started by Jesse Mostipak](https://twitter.com/kierisi/status/1113945904698892289), with contributions from [Chris Engelhardt](https://twitter.com/EngelhardtCR), [Laura Ación](https://twitter.com/_lacion_/status/1113960425174052864), [Bruna Wunderwald](https://twitter.com/bwundervald/status/1114907322441244672), [Erin LeDell](https://twitter.com/ledell/status/1115138789293211649).
```

File: content/post/statement-about-datacamp.Rmd (hunk 4 above)
Comment (maelle) [reply content/post/statement-about-datacamp.Rmd]: Approved, but can one do that, legally?

**Comment by chucheria** [comment]
Sorry I merged before, it seems it wasn't ready. I can't open the pr again. @maelle could you open one? thank you and sorry again. Please @ledell @gdequeiroz merge when you are ready.

**Pull request merged**